addon/maptools/measuretool.js

```diff
@@ -0,0 +1,39 @@
+import MapTool from 'ember-flexberry-gis/maptools/maptool';
+
+export default MapTool.extend({
+  _layer: null,
+  _measure: null,
+
+  isMultiTool() {
+    return true;
+  },
+
+  measureClose(e) {
```

Comment:
Do you mean _layer, _measure
Deleted on disable()?
Deleted - set to null or undefined?